### ISSUE TITLE
Did this stop working december 2021

### ISSUE THREAD

**Issue by looolz** [YR/MMM-YrNow#20]
Suddenly stopped working for me. It just says loading.

**Comment by Sti2nd** [comment]
Same here

**Comment by Sti2nd** [comment]
Yup! There is a fix for this here https://github.com/YR/MMM-YrNow/pull/19

**Comment by sondrehoglund** [comment]
Does this works for you guys? I tried changing the URL as stated, but it won´t load
